### PR TITLE
Fix codecov

### PR DESCRIPTION
**Pull request by carllerche**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,12 +34,6 @@ script:
   # Generate docs
   - cargo doc --no-deps
 
-after_success:
```
Comment (hawkw) [inline .travis.yml]: Oh, my bad, sorry.

**Comment by hawkw** [comment]
Looked into this by running `travis-cargo coverage` on my machine locally, and it looks like `travis-cargo` may be no longer passing the correct flags to `kcov`?

**Comment by hawkw** [comment]
@carllerche I think my PR #61 will _actually_ fix coverage report uploading from CI, sorry this was such a struggle!

If we want GitHub commit status for coverage reports, someone who's a repo owner will have to [set up that integration](https://github.com/apps/codecov).

**Comment by carllerche** [comment]
I'll close this in favor of #62

**Comment by hawkw** [comment]
FWIW we're not the only ones having issues w/ `travis-cargo`: https://github.com/huonw/travis-cargo/issues/72
unfortunately, it looks pretty unmaintained.

**Pull request closed without merging**